Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Makie"
 uuid = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
 authors = ["Simon Danisch", "Julius Krumbiegel"]
-version = "0.15.0"
+version = "0.15.1"
```

Comment:
Best to leave this out, since with the backends & mono repo, things got a bit more complicated ;) 
We want to tag the current master anyways, so just leave it to us :)